cookbook/recipes/common.rb

```diff
@@ -0,0 +1,19 @@
+node.set[:java][:jdk_version] = 7
+
+include_recipe 'java'
+include_recipe 'baragon::build'
+
+%w(/etc/baragon /var/log/baragon).each do |dir|
```

Comment:
I find it's best to put things like this each on their own line, in order to have cleaner diffs as time goes on.